### PR TITLE
Refactoring platform services repo

### PR DESCRIPTION
**Pull request by SiniWilson**

### CODE REVIEW

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh [hunk 1]
```diff
@@ -14,29 +14,24 @@ sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack
 
 #-------------------------------------------
 
-platform_services=("cognito-authorizer" "platform_logs" "platform_usermanagement" "platform-services-handler" "platform_events" "platform_services" "platform_logout" "platform_login" "cloud-logs-streamer" "is-service-available" "delete-serverless-service" "create-serverless-service" "platform_email" "platform_events-handler" )
+platform_services=("cognito-authorizer" "logs" "usermanagement" "services-handler" "events" "services" "logout" "login" "cloud-logs-streamer" "is-service-available" "delete-serverless-service" "create-serverless-service" "email" "events-handler" )
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: Have the actual dir name and eventual repo name so its consistent and easier to add in the two places (this file plus scmpush)

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh [hunk 2]
```diff
@@ -14,29 +14,24 @@ sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack
...
   uuid=`uuidgen -t`
-  echo -n > ./jazz-core/$element/deployment-env.yml
-  echo "service_id: "$uuid >> ./jazz-core/$element/deployment-env.yml
- 
-  if [[ $element =~ ^platform ]] ; then
-    service_name="${element:9}"
-  else
-    service_name=$element
-  fi
+  echo -n > ./jazz-core/jazz_$element/deployment-env.yml
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: Once you maker the change suggested above, you can remove jazz_ here

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh (hunk 1 above)
Comment (SiniWilson) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: If I add the actual repo name over here I have to split it get the service name. Please advice

Comment (SiniWilson) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: In cooperated the review comment

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh (hunk 2 above)
Comment (SiniWilson) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: In cooperated the review comment

**Pull request merged**